moveit_setup_assistant/src/widgets/perception_widget.cpp

```diff
@@ -64,7 +64,7 @@ PerceptionWidget::PerceptionWidget(QWidget* parent, const MoveItConfigDataPtr& c
       new HeaderWidget("Setup 3D Perception Sensors",
                        "Configure your 3D sensors to work with MoveIt "
                        "Please see <a "
-                       "href='http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/perception_pipeline/"
+                       "href='https://ros-planning.github.io/moveit_tutorials/doc/perception_pipeline/"
```

Comment:
This link doesn't work?

Reply:
This link is split between two lines, similarly to the previous link. Full link is this link + the next line (https://ros-planning.github.io/moveit_tutorials/doc/perception_pipeline/perception_pipeline_tutorial.html)

Reply:
lol, got it